notebooks/text-to-image-genai/README.md

```diff
@@ -9,8 +9,6 @@ The Hugging Face Optimum Intel API is a high-level API that enables us to conver
 
 ## Notebook Contents
 
```

Comment:
```suggestion
This notebook demonstrates how to perform image generation  using OpenVINO. GenAI Text2ImagePipeline
```

Reply:
then better to move it to this section.

The first section is just intro to notebook
Notebook content section is more detailed description what notebook do. Having only table of content in it looks slightly rude, need some introduction sentence before it

Reply:
Moved